guides/source/form_helpers.md

```diff
@@ -621,6 +621,85 @@ The first parameter specifies which value should be selected and can either be a
 
 will produce the same output and the value chosen by the user can be retrieved by `params[:date][:year]`.
 
+Creating Checkboxes for Has Many relations
+------------------------------------------
+
+Generating checkboxes options that act appropriately with Active Record models and strong parameters can be a confusing undertaking. Luckily there is a helper function to significantly reduce the work required.
```

Comment:
All new guides should be wrapped to 80 characters.

Reply:
I text wrapped the text but not the code blocks, I imagine this is what is desired?